pkg/controller/deployment/deployment_controller.go

```diff
@@ -487,13 +487,13 @@ func (dc *DeploymentController) handleErr(err error, key interface{}) {
 	}
 
 	if dc.queue.NumRequeues(key) < maxRetries {
-		klog.V(2).InfoS("Error syncing deployment", "deployment", klog.KRef(ns, name), "err", err)
+		klog.V(2).InfoS("Error syncing deployment", "deployment", klog.KRef(ns, name), "key", key, "err", err)
```

Comment:
`key` can be inferred from `deployment` making it redundant. 
```suggestion
		klog.V(2).InfoS("Error syncing deployment", "deployment", klog.KRef(ns, name), "err", err)
```

Reply:
removed `key`.